spec/fixtures/modules/results/lib/puppet/functions/results/make_apply_result.rb

```diff
@@ -8,7 +8,9 @@
   end
 
   def create(uri, value)
-    target = Bolt::Target.new(uri)
+    inventory = Puppet.lookup(:bolt_inventory)
+    inventory.get_target(uri)
+    target = Bolt::Target.new(uri, inventory)
```

Comment:
`inventory.get_target(uri)` should already be returning a `Bolt::Target` instance